frontend/scss/_bootstrap-variables.scss

```diff
@@ -0,0 +1,9 @@
+﻿@import "../node_modules/bootstrap/scss/variables";
+
+$font-family-base: "Montserrat";
+
+$headings-font-weight: 700;
+$h1-font-size: $font-size-base * 3.125;
```

Comment:
❓ I think using h1-h6 primitives does not fit the apps context. Perhaps instead of modifying h1-h6 use semantic css classes in contexture?

Reply:
When I opened a domain there is an apparent structure (title, section with title, subsection with title) and in my head I went h1->h2->h3, but since it isn't a document but rather a collection of components I agree that it makes sense to use semantic CSS there. In case we need text pages we could still use h1-h6, but let's first see if we need it.